rskj-core/src/test/java/co/rsk/peg/PegoutCreationEntryTest.java

```diff
@@ -0,0 +1,74 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.crypto.Keccak256;
+import org.apache.commons.lang3.RandomUtils;
```

Comment:
Let's not add any more dependencies to this package. I think there was a plan to remove it. Just used a fixed seed value when calling createHash method